Src/Zle/zle_refresh.c

```diff
@@ -502,7 +511,15 @@ set_region_highlight(UNUSED(Param pm), char **aval)
 	while (inblank(*strp))
 	    strp++;
 
-	match_highlight(strp, &rhp->atr);
+	strp = (char*) match_highlight(strp, &rhp->atr);
+
+	while (inblank(*strp))
+	    strp++;
+
+	if (strpfx("owner=", strp))
+	    rhp->owner = ztrdup(strp + 6);
```

Comment:
Given the above comment, this shouldn't copy the entire rest of the string, but only up to the next space, for extensibility (forward compatibility).

Reply:
Should be resolved by the just-pushed commits.  (The hex commit id's will be changed later when I squash them, so I'm not pointing to a specific commit.)

Reply:
New tests added for this.